Documentation: Updates for "Verification results" (#979)
model-checking/kani: docs/src/verification-results.md

# Verification results

Running Kani on a harness produces an output that includes a set of checks as
follows:

```
RESULTS:
Check 1: example.assertion.1
         - Status: <status>
         - Description: <description>
         - Location: <location>
[...]
```

> In short, Kani determines the verification result for the harness based on the

Reviewer: ```suggestion
Kani determines the verification result for the harness based on the
```

> result (i.e., `<status>`) from these checks (also known as "properties"). If all

Reviewer: ```suggestion
result (i.e., `<status>`) of each individual check (also known as "properties"). If all
```

> checks are successful then its verification result is successful. Otherwise the

Reviewer: ```suggestion
checks are successful then the overall verification result of the harness is successful. Otherwise the
```

> verification is failed, which indicates issues with the code under verification.

Reviewer: ```suggestion
verification fails, which indicates issues with the code under verification.
```
